### ISSUE TITLE
[BUG] Incorrect numbering for plenty of items on my Steam wishlist

### ISSUE THREAD

**Issue by przemo20** [IsThereAnyDeal/AugmentedSteam#2096]
### Expected Behavior
All the items on a wishlist have the correct index number (numbering).

### Actual Behavior
When I view [my Steam wishlist](https://store.steampowered.com/wishlist/id/przemo20/) _(little warning: there are a lot of NSFW titles there)_ and – for example – I sort it by the price, I see a lot of zeros and “undefined” in the left column with numbering _(it's been for a while like that; it's not a new issue)_.

For example, here's how the entry for The Dark Pictures Anthology: House of Ashes looks when I sort it by price _(AS numbering is displayed)_:
![obraz](https://github.com/user-attachments/assets/023f2846-f65f-4351-93cb-2cb37e1cf771)

But when I switch it to “Your rank” _(Steam numbering is displayed)_, it has an index of 171:
![obraz](https://github.com/user-attachments/assets/8ec719aa-c797-4587-9e2a-c54caa78c5fd)

...and just noticed that Little Hope shows here with an index of 108 (so the entire numbering done by AS might be incorrect?):
![obraz](https://github.com/user-attachments/assets/80359529-be14-4921-b9fd-f811923dcff7)

And here are two examples of undefined (respectively no. 305 and no. 239 when I switch sorting to “Your rank”):
![obraz](https://github.com/user-attachments/assets/7df4070c-b756-4cbe-8032-866b51b1bc18)

I don't have an idea what might be causing this – is this because of having some titles in the wishlist that got delisted/hidden (like the [Deadpool game](https://steamdb.info/app/224060/)) and AS is getting confused? Is there anything I could provide that could help you debug this?

Here's an exported JSON files with my wishlist _(maybe that will help)_: [wishlist.json](https://github.com/user-attachments/files/18186678/wishlist.json)

### Steps to Reproduce the Problem

  1. ??? _(Unsure how to reproduce this, other than by logging to my Steam account, since you can only see the numbering on your own wishlist, not others)_
  2. ???
  3. ???

### Specifications

  - Augmented Steam Version: 4.2.1
  - Browser Version: Mozilla Firefox 133.0.3 (64-bit)

**Comment by candela97** [comment]
Yeah, this is limitation due to the new wishlist. You need to load the whole wishlist without any filters first for this feature to work properly, but that might not always be possible. On my end it always defaults to `min_discount=any` for whatever reason...

**Comment by Pamlau** [comment]
is there any way to disable/hide the numbering itself?

**Comment by tfedor** [comment]
We don't do custom ordering, never did, we're just showing rank in all search options. Now that I'm thinking about it, I'm not really sure if there's any value in that.

**Comment by SoraTakai** [comment]
It doesn't serve any value; or actually its effect is purely negative since it's being shown for nothing.

Could you please just get rid of them?

**Comment by ojonquet** [comment]
Always showing the rank under all search filters allowed me to easily reorder my very big wishlist. 

But now, in my specific case, modifying the rank of a previously added game has become extremely tedious (because my only option is to first find it manually by scrolling through the "rank" order). 

Even adding a new game to the top is not really user-friendly anymore. 

Or am I missing something?

If not, please add this neat feature back!
